bundle.Dockerfile

```diff
@@ -13,9 +13,9 @@ LABEL name="openshift4-wincw/windows-machine-config-operator-bundle" \
 # delivered via an index image
 LABEL com.redhat.delivery.operator.bundle=true
 
-# This second label tells the pipeline which versions of OpenShift the operator supports (4.6+).
+# This second label tells the pipeline which versions of OpenShift the operator supports (4.6).
 # This is used to control which index images should include this operator.
-LABEL com.redhat.openshift.versions="v4.6"
+LABEL com.redhat.openshift.versions="=v4.6"
```

Comment:
Please mention this in the commit message and why we are making this change.

Reply:
this can be added as a different commit as it is not tied to version change.

Reply:
@VaishnaviHire even with a different commit the PR header should be changed.